### ISSUE TITLE
Upgrade from Chromium 98.0.4758.66 to Chromium 98.0.4758.72.

### ISSUE THREAD

**Issue by mkarolin** [brave/brave-browser#20726]
Minor Chromium bump.

https://chromium.googlesource.com/chromium/src/+log/98.0.4758.66..98.0.4758.72?pretty=fuller&n=10000

QA tests:

Check branding items
Check for version bump

Additional checks:

No specific code changes in Brave (only line number changes in patches).

**Comment by kjozwiak** [comment]
Remaining items:

* `Win` & `Android ARM` need to run through the SKU cases as per https://github.com/brave/brave-browser/issues/19664#issuecomment-1022821716
* `macOS` & `Linux` need to run through the areas mentioned via https://github.com/brave/brave-browser/issues/19664#issue-1060511172 and the SKU cases as per https://github.com/brave/brave-browser/issues/19664#issuecomment-1022821716.

**Comment by srirambv** [comment]
Verification passed on the following devices running 1.35.97 x64 build

Oppo Reno 5 (Android 11) | Samsung Tab A (Android 10)
-- | --
<img width=300 src=https://user-images.githubusercontent.com/17010094/151553846-71e0ece0-3c29-4d28-84dd-94ac1e5c30a9.png> | <img width=400 src=https://user-images.githubusercontent.com/17010094/151553991-d499cdfa-985e-4219-a5ff-f095ab846b88.png> 


Pending Android verification https://github.com/brave/brave-browser/issues/19664#issuecomment-1022821716. Will add Android pass labels once the SKU verification is done

**Comment by stephendonner** [comment]
Verification PASSED on

Brave | 1.35.97 Chromium: 98.0.4758.72  (Official Build)  (x86_64)
--|--
Revision |d63e30f73734c5326037b0d94eceed77168d95e1-refs/branch-heads/4758@{#905}
OS | macOS Version 11.6.1 (Build 20G224)

1. **_Settings UI_**: 
- Confirmed all the settings are displayed as expected in `brave://settings` and UI also looks good.
- Confirmed content settings are displayed as expected in `brave://settings/content`
- Confirmed security settings are as expected, no new settings are added as part of CR 98 in `brave://settings/security`
- Confirmed `brave://settings/searchEngines` is displayed as expected no new settings added, the UI looks good
- Confirmed all the settings under `brave://settings/extensions` are displayed as expected
- Confirmed `Share this page` options works as expected



2.  **_Brave News_**
- Confirmed clicking on `Show Brave News` shows the Brave News feeds
- Confirmed content cards are loading the images correctly in feeds when the Brave News feed is scrolled down
- Confirmed click on any image/link on news feed redirects to correct target link
- Confirmed images render properly on brave news feeds when any publisher is enabled/disabled via `Sources`

3. **_Components updater_**
- Testing is covered in https://github.com/brave/brave-browser/issues/20668#issuecomment-1021837180

4. **_Permissions bubble shows correctly_**
- Confirmed site settings permissions work as expected

Example | Example | Example
---|----|---
 |  | 

5. **_Cosmetic filtering_**

- Cosmetic filtering works as expected
- Element on any page can be blocked via block element selector 

element blocked | blocked rule | remove rules | blocked elements reappeared
----|-----|----|-----
| |  | 

6. **_AdBlock regional list_**

- Encountered the issue https://github.com/brave/brave-browser/issues/4367
- Launched browser using `brave.exe --lang=fr` and confirmed correct Adblock regional filter is pre-selected in brave://adblok
- Launched browser using `brave.exe --lang=de` and confirmed correct Adblock regional filter is pre-selected in brave://adblok

Example | Example
-----|------
 | 

7. **_Tabs search button toggle in settings works correctly_** 
- Confirmed Tab search option is shown when the `Show tab search button` settings is enabled
- Confirmed able to search the particular tab from list of tabs
- Confirmed Tab search option isn't shown when the `Show tab search button` settings is disabled
- Confirmed `Show tab search button` setting retained after browser restart

Example | Example | Example
----|-----|-----
 | |

**Comment by stephendonner** [comment]
## Verification `PASSED` on 

Brave | 1.35.97 Chromium: 98.0.4758.72 (Official Build) (64-bit) 
--|--
Revision | d63e30f73734c5326037b0d94eceed77168d95e1-refs/branch-heads/4758@{#905}
OS | Linux

<img width="1358" alt="Screen Shot 2022-01-28 at 2 06 38 PM" src="https://user-images.githubusercontent.com/387249/151632634-2929c975-c3ff-489c-b4b4-541366daa770.png">



<summary>1. Settings UI: </summary>
<details>

- Confirmed all the settings are displayed as expected in `brave://settings` and UI also looks good.
- Confirmed content settings are displayed as expected in `brave://settings/content`
- Confirmed security settings are as expected, no new settings are added as part of CR 98 in `brave://settings/security`
- Confirmed `brave://settings/searchEngines` is displayed as expected no new settings added, the UI looks good
- Confirmed all the settings under `brave://settings/extensions` are displayed as expected
- Confirmed `Share this page` options works as expected


Encountered https://github.com/brave/brave-browser/issues/20750 while testing.

</details>

<summary>2.  Brave News</summary>
<details>

- Confirmed clicking on `Show Brave News` shows the Brave News feeds
- Confirmed content cards are loading the images correctly in feeds when the Brave News feed is scrolled down
- Confirmed click on any image/link on news feed redirects to correct target link
- Confirmed images render properly on brave news feeds when any publisher is enabled/disabled via `Sources`

</details>

<summary>3. Components updater</summary>
<details>

Compared `brave://components` to `release`

`brave://components` | `release`
-----------------------|-------

</details>

<summary>4. Permissions bubble shows correctly</summary>
<details>

- Confirmed site settings permissions work as expected

Example | Example | Example | Example
---|----|-----|-----


</details>

<summary>5. Cosmetic filtering</summary>
<details>

- Cosmetic filtering works as expected
- Element on any page can be blocked via block element selector 

element blocked | blocked rule | remove rules | blocked elements reappeared
----|-----|----|-----

</details>

<summary>6. AdBlock regional list</summary>
<details>

- Encountered the issue https://github.com/brave/brave-browser/issues/4367
- Launched browser using `--lang=fr` and confirmed correct Adblock regional filter is pre-selected in `brave://adblock`
- Launched browser using `--lang=de` and confirmed correct Adblock regional filter is pre-selected in `brave://adblock`

Example | Example
-----|------
 
</details>

<summary>7. Tabs search button toggle in settings works correctly</summary>
<details>

- Confirmed Tab search option is shown when the `Show tab search button` settings is enabled
- Confirmed able to search the particular tab from list of tabs
- Confirmed Tab search option isn't shown when the `Show tab search button` settings is disabled
- Confirmed `Show tab search button` setting retained after browser restart

Example | Example | Example
----|-----|-----


</details>

<summary>8. SKUs</summary>
<details>

Followed the steps for the two SKU testcases over in https://github.com/brave/brave-core/pull/11985#issue-1114295405

Case 1: Verify SKU functionality is disable on allow-list domains
1. Visit https://account.brave.software and enter basic auth credentials (reach out to @bsclifton for details)
2. Open dev tools (F12, etc) and go to `Console` tab
3. Paste in: `await window.chrome.braveSkus.refresh_order('0ee1e3ee-afc3-4f2f-b2a2-3d1e401e1530')` and hit enter. It should show
    ```
    Uncaught TypeError: Cannot read properties of undefined (reading 'refresh_order')
    at <anonymous>:1:31
    ```
4. Steps 1-3 can be repeated on `https://account.brave.com` or `https://account.bravesoftware.com`

`account.brave.com` | `account.bravesoftware.com` | `account.brave.software`
--------------------|-----------------------------|------------------------



Case 2: Verify SKU functionality is disabled on non-allowed domains
1. Visit any non-Brave website
2. Open dev tools (F12, etc) and go to `Console` tab
3. Paste in: `await window.chrome.braveSkus.refresh_order('0ee1e3ee-afc3-4f2f-b2a2-3d1e401e1530')` and hit enter. It should show
    ```
    Uncaught TypeError: Cannot read properties of undefined (reading 'refresh_order')
    at <anonymous>:1:31
    ```


</details>

**Comment by LaurenWags** [comment]
labelling `QA/No` and `release-notes/exclude` as this is superseded by https://github.com/brave/brave-browser/issues/20741
